doc/userguide/src/ExtendingRobotFramework/CreatingTestLibraries.rst

```diff
@@ -1189,16 +1189,19 @@ messages, specify the log level explicitly by embedding the level into
 the message in the format `*LEVEL* Actual log message`, where
 `*LEVEL*` must be in the beginning of a line and `LEVEL` is
 one of the available logging levels `TRACE`, `DEBUG`,
-`INFO`, `WARN`, `FAIL` and `HTML`.
+`INFO`, `WARN`, `ERROR` and `HTML`.
 
-Warnings
-''''''''
+Errors and Warnings
+'''''''''''''''''''
 
-Messages with `WARN` level are automatically written into `the
-console and into separate Test Execution Errors section`__ in log
-files. This makes warnings more visible than other messages and allows
+Messages with `ERROR` or `WARN` level are automatically written to the
+console and a separate `Test Execution Errors section`__ in the log
+files. This makes errors more visible than other messages and allows
 using them for reporting important but non-critical problems to users.
 
+.. note:: Use of the `ERROR` log level is new in Robot 2.9.
```

Comment:
ERROR level itself isn't new in 2.9. Should probably be something like "Automatically adding ERRORs logged by keywords to the Test Execution Errors section is new in Robot Framework 2.9.".